<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titlebar</class>
 <widget class="QWidget" name="titlebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleWidget {
    border-bottom: 1px solid rgb(235, 235, 235);
    background-color: rgb(250, 245, 245);
}
#icon{
	border: 1px solid rgb(235, 235, 235);
}
#title {
    font-family: &quot;微软雅黑&quot;;
    font: 10pt;
    background-color: transparent;
    color: rgb(60, 60, 60);
}
QPushButton {
    font-family: &quot;微软雅黑&quot;;
    font-size: 10pt;
    color: rgb(60, 60, 60);
	border:1px solid transparent;
    border-radius: 4px;
}
QPushButton::icon {
    padding-right: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="icon">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap>:/images/icon/icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title">
        <property name="text">
         <string>d3dxModManager</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="titleSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnMin">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../../../resource/resources.qrc">
          <normaloff>:/iamge_pack/title_bar/minimize.png</normaloff>:/iamge_pack/title_bar/minimize.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMax">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../../../resource/resources.qrc">
          <normaloff>:/iamge_pack/title_bar/maximize.png</normaloff>
          <normalon>:/iamge_pack/title_bar/windowed.png</normalon>:/iamge_pack/title_bar/maximize.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../../../resource/resources.qrc">
          <normaloff>:/iamge_pack/title_bar/shutdown.png</normaloff>:/iamge_pack/title_bar/shutdown.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resource/resources.qrc"/>
 </resources>
 <connections/>
</ui>
